frontend/src/routes/(authenticated)/myaccount/+page.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
+    // This script handles the account settings page.
+    // For now, it only allows you to modify the users display name
+    // Minimal changes allow it to modify the users email as well (but this should have a re-verification system)
+    import type { PageData } from './$types';
+    import t from '$lib/i18n';
+    import { Button, Form, Input, lexSuperForm } from '$lib/forms';
+    import { Page } from '$lib/layout';
+    import { user } from '$lib/user';
+    import {_changeUserAccountData} from "./+page";
+    import type {ChangeUserAccountDataInput} from '$lib/gql/types';
+
+    // Get users data (reactive)
+    $: users_name = $user?.name; // not to be confused with username
+    $: email = $user?.email;
+    $: userid = $user?.id;
+    let newName: string = users_name || '';
```

Comment:
I think you should probably have a `newEmail` as well.

If you prefer, you could use `let newUser = {email: '', name: ''}`, although with a `lexSuperForm` we would use the form store instead of needing a local field.